### PR TITLE
Fixed FluidCooler:TwoSpeed fatal error with Low Speed Nominal Capacity=autocalulate

### PR DESCRIPTION
**Pull request by JasonGlazer**
Addresses GitHub #4694  including new unit test

### CODE REVIEW

**Comment by JasonGlazer** [comment]
The build status warnings were due to a stray semicolon so that was removed and retested and I merged it with develop as long as I was touching the code again.

File: src/EnergyPlus/FluidCoolers.cc [hunk 1]
```diff
@@ -635,6 +531,160 @@ namespace FluidCoolers {
...
+	)
+	{
+		// FUNCTION INFORMATION:
+		//       AUTHOR:          Chandan Sharma
+		//       DATE WRITTEN:    August 2008
+		//       MODIFIED         Chandan Sharma, FSEC, April 2010
+		//       RE-ENGINEERED    Jason Glazer, GARD Analytics, February 2015, refactor into a separate function
+
+		// PURPOSE OF THIS FUNCTION:
+		// Separate the testing of inputs related to design so that it could be called from the unit tests
```
Comment (jasondegraw) [inline src/EnergyPlus/FluidCoolers.cc]: @JasonGlazer Same thing here as the other pull request. Are you OK with this info?

Comment (JasonGlazer) [inline src/EnergyPlus/FluidCoolers.cc]: Thanks again

**Comment by JasonGlazer** [comment]
@jasondegraw Hopefully merging will let these test cases pass. Is that my responsibility or yours?

**Pull request merged**